tests/pkg_keccak/tests/01-run.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2017 Freie Universität Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import sys
+from testrunner import run
+
+
+def testfunc(child):
+    child.expect('OK \(\d+ tests\)')
```

Comment:
```suggestion
    child.expect(r'OK (\d+ tests)')
```

To fix the errors reported by Travis

Reply:
(I don't know if the `pkg_hacl` test is passing btw...

Reply:
Many tests where not updated as the `linter` is only on newly added files.
It was found just before the release I think but did not have time to update the scripts yet.